com.valvesoftware.Steam.yml

```diff
@@ -75,6 +75,12 @@ add-extensions:
     download-if: active-gl-driver
     enable-if: active-gl-driver
 
+  com.valvesoftware.Steam.CompatibilityTool:
+    subdirectories: true
+    directory: compatibilitytools.d
```

Comment:
Multiple 3rd-partiy compatibility tools install themselves to `$prefix/share/steam/compatibilitytools.d`. Maybe we should do the same?

Reply:
I haven't actually tried these so I'm going to trust others on this for now. I guess that means when you create an extension, you would drop files into the root directory.

Reply:
Well, I guess the mountpoint could be changed later anyway.